src/strtab.rs

```diff
@@ -133,36 +148,32 @@ impl<'a> Index<usize> for Strtab<'a> {
 
 #[test]
 fn as_vec_no_final_null() {
-    let bytes = b"\0printf\0memmove\0busta";
-    let strtab = unsafe { Strtab::from_raw(bytes.as_ptr(), bytes.len(), 0x0) };
```

Comment:
thank you for removing the unsafe tests here; probably should at some point add tests for the unsafe api, but here the unsafety was not super important for testing basic functionality